device/iot-device-samples/send-event-x509/src/main/java/samples/com/microsoft/azure/sdk/iot/SSLContextBuilder.java

```diff
@@ -143,128 +152,4 @@ public static RSAPrivateKey parsePrivateKeyString(String privateKeyPEM) throws I
 
         return randomChars;
     }
-
-    private static TrustManagerFactory generateTrustManagerFactory(KeyStore trustKeyStore)
```

Comment:
This logic was to build the TrustManagers out of the Iot hub public certs from source, which we no longer want users to do.